rmw_implementation/test/benchmark/benchmark_symbols.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <memory>
+
+#include "performance_test_fixture/performance_test_fixture.hpp"
+
+#include "../../src/functions.cpp"
+
+using performance_test_fixture::PerformanceTest;
+
+BENCHMARK_F(PerformanceTest, prefetch_symbols)(benchmark::State & st)
+{
+  for (auto _ : st) {
+    std::shared_ptr<rcpputils::SharedLibrary> lib = load_library();
+    prefetch_symbols();
```

Comment:
```suggestion
    prefetch_symbols();
```

`load_library` doesn't cache the library as a global, so when `prefetch_symbols` runs, the library is loaded again by `get_library`. So this explicit load is extra.

Reply:
In the interest of moving this PR along, I addressed this with 0507fc2.